backend/app/views/spree/admin/promotions/_activations_new.html.erb

```diff
@@ -0,0 +1,66 @@
+<% activation_type = params[:activation_type] || 'single_code' %>
+<div class="row" id="js_promotion_activation">
```

Comment:
@jhawthorn why you are not using dash but underscores to separate words for all elements ids?